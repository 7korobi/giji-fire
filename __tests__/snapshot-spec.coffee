{ mount } = require '@vue/test-utils'
Vue = require 'vue'
Vuex = require 'vuex'
Vue.use Vuex

snap = ({ module, name, propsData, store })->
  describe name, =>
    test 'snapshot', =>
      wrapper = mount module, { propsData, store }
      expect( wrapper.html() ).toMatchSnapshot()

snap
  name: 'AppLogo'
  module: require '~/components/AppLogo.vue'

snap
  name: 'AFooter'
  module: require '~/components/a-footer.vue'

snap
  name: 'AHeader'
  module: require '~/components/a-header.vue'
  propsData:
    title: "人狼議事"

snap
  name: 'BreadCrumb'
  module: require '~/components/bread-crumb.vue'

snap
  name: 'BtnMarker'
  module: require '~/components/btn-marker.vue'
  store: new Vuex.Store
    firebase:
      user:
        uid: "TEST-UID"
  propsData:
    back_url:
      query:
        idx: "test-1-top"
    write_at: 1
    log: {}

snap
  name: 'Btn 1'
  module: require '~/components/btn.vue'
  propsData:
    as: 10
    value: 5

snap
  name: 'Btn 1'
  module: require '~/components/btn.vue'
  propsData:
    as: [10]
    value: [5]
    bool: 'include'

snap
  name: 'ChatFoot'
  module: require '~/components/chat-foot.vue'
  store: new Vuex.Store
    menu:
      side:
        shows: []
  propsData:
    anker: "*TEST*"

snap
  name: 'ChatHead sign'
  module: require '~/components/chat-head.vue'
  propsData:
    full: true
    head: "おなまえ"
    sign: "7korobi"

snap
  name: 'ChatHead to'
  module: require '~/components/chat-head.vue'
  propsData:
    head: "おなまえ"
    to:   "おあいて"
