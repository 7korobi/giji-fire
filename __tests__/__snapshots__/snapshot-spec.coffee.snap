// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AFooter snapshot 1`] = `
<div
  class="chat report footer"
>
  <!---->
  <!---->
  <quill-view
    class="text giji"
    value=""
  />
  <div
    class="text giji"
  >
    <article>
      人狼議事キャラセット by りりんら
管理
      <q
        class="btn"
      >
        sol・la
      </q>
      <a
        class="btn"
        href="mailto:7korobi@gmail.com"
        title="mailto:7korobi@gmail.com"
      >
        ななころび
      </a>
      <div
        class="copyright"
      >
        <p>
          下記の場所以外では、人狼議事内キャラチップ
の利用を許諾しておりません。ご了承ください。
        </p>
      </div>
      <div
        class="copyright"
      >
        <p>
          議事総合トップ
人狼議事lobby
人狼議事morphe
人狼議事cafe
人狼議事perjury
人狼議事xebec
人狼議事crazy
人狼議事ciel

SoyBean
Sangria
@7korobi
        </p>
      </div>
    </article>
  </div>
  <chat-foot />
</div>
`;

exports[`AHeader snapshot 1`] = `
<div>
  <no-ssr>
    <welcome
      title="人狼議事"
      top="0"
    >
      <div
        class="btns form"
      >
        <span
          class="zoom"
        >
          <btn
            as="1.0"
          >
            １
          </btn>
          <btn
            as="0.75"
          >
            ¾
          </btn>
          <btn
            as="0.5"
          >
            ½
          </btn>
        </span>
        <span
          class="font"
        >
          <btn
            as="large"
          >
            大判
          </btn>
          <btn
            as="novel"
          >
            明朝
          </btn>
          <btn
            as="std"
          >
            ゴシック
          </btn>
          <btn
            as="small"
          >
            繊細
          </btn>
        </span>
        <span
          class="theme"
        >
          <btn
            as="cinema"
          >
            煉瓦
          </btn>
          <btn
            as="pop"
          >
               噴出
          </btn>
          <btn
            as="snow"
          >
              雪景
          </btn>
          <btn
            as="star"
          >
              蒼穹
          </btn>
          <btn
            as="night"
          >
             闇夜
          </btn>
          <btn
            as="moon"
          >
              月夜
          </btn>
          <btn
            as="wa"
          >
              和の国
          </btn>
        </span>
      </div>
    </welcome>
  </no-ssr>
  <div
    class="outframe filmend-frame"
  >
    <div
      class="contentframe"
    >
      <div
        class="filmend"
      />
    </div>
  </div>
</div>
`;

exports[`AppLogo snapshot 1`] = `
<div
  class="VueToNuxtLogo"
>
  <div
    class="Triangle Triangle--two"
  />
   
  <div
    class="Triangle Triangle--one"
  />
   
  <div
    class="Triangle Triangle--three"
  />
   
  <div
    class="Triangle Triangle--four"
  />
</div>
`;

exports[`BreadCrumb snapshot 1`] = `
<ol
  class="bread-crumb"
>
  <li>
    <nuxt-link
      to="/"
    >
      Top
    </nuxt-link>
  </li>
</ol>
`;

exports[`Btn include off snapshot 1`] = `
<a
  class="btn"
/>
`;

exports[`Btn include on snapshot 1`] = `
<a
  class="btn"
/>
`;

exports[`Btn off snapshot 1`] = `
<a
  class="btn"
/>
`;

exports[`Btn on snapshot 1`] = `
<a
  class="btn active"
/>
`;

exports[`BtnMarker snapshot 1`] = `
<nuxt-link
  class="item active"
  replace="replace"
  to="[object Object]"
>
  <i
    class="mdi mdi-map-marker"
  />
  
</nuxt-link>
`;

exports[`Chat 1 snapshot 1`] = `
<table
  class="talk"
  id="test-1-1-ss-1"
  show="talk"
>
  <tbody>
    <tr>
      <th>
        <portrate
          face_id="c10"
        />
      </th>
      <td>
        <div
          class="baloon SSAY"
        />
        <div
          class="chat SSAY"
        >
          <chat-head
            head="自由記入 ゾーイ"
            to="宛先"
          />
          <hr />
          <quill-view
            class="text quill"
            value="テスト<strong>文字列</strong>"
          />
          <!---->
          <chat-foot
            anker=""
          />
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Chat 1 snapshot 2`] = `
<table
  class="talk"
  id="test-1-1-ss-1"
  show="talk"
>
  <tbody>
    <tr>
      <th>
        <portrate
          face_id="c10"
        />
      </th>
      <td>
        <div
          class="baloon SSAY"
        />
        <div
          class="chat SSAY"
        >
          <chat-head
            head="自由記入 ゾーイ"
            to="宛先"
          />
          <hr />
          <quill-input
            class="text quill"
            part_id="test-1-1"
            phases=""
          />
          <!---->
          <chat-foot
            anker=""
          />
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Chat 2 snapshot 1`] = `
<table
  class="talk"
  id="test-1-1-ss-2"
  show="talk"
>
  <tbody>
    <tr>
      <th>
        <portrate
          face_id="c10"
        />
      </th>
      <td>
        <div
          class="baloon SSAY"
        />
        <div
          class="chat SSAY"
        >
          <chat-head
            head="自由記入 ゾーイ"
          />
          <hr />
          <quill-view
            class="text quill"
            value="テスト<strong>文字列</strong>"
          />
          <!---->
          <chat-foot
            anker=""
          />
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Chat 2 snapshot 2`] = `
<table
  class="talk"
  id="test-1-1-ss-2"
  show="talk"
>
  <tbody>
    <tr>
      <th>
        <portrate
          face_id="c10"
        />
      </th>
      <td>
        <div
          class="baloon SSAY"
        />
        <div
          class="chat SSAY"
        >
          <chat-head
            head="自由記入 ゾーイ"
          />
          <hr />
          <quill-input
            class="text quill"
            part_id="test-1-1"
            phases=""
          />
          <!---->
          <chat-foot
            anker=""
          />
        </div>
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Chat 3 snapshot 1`] = `
<div
  class="chat post SSAY"
  id="test-1-1-ss-3"
  show="post"
>
  <chat-head
    head="自由記入 ゾーイ"
  />
  <hr />
  <quill-view
    class="text quill"
    value="テスト<strong>文字列</strong>"
  />
  <!---->
  <chat-foot
    anker=""
  />
</div>
`;

exports[`Chat 3 snapshot 2`] = `
<div
  class="chat post SSAY"
  id="test-1-1-ss-3"
  show="post"
>
  <chat-head
    head="自由記入 ゾーイ"
  />
  <hr />
  <quill-input
    class="text quill"
    part_id="test-1-1"
    phases=""
  />
  <!---->
  <chat-foot
    anker=""
  />
</div>
`;

exports[`Chat 4 snapshot 1`] = `
<div
  class="chat report SSAY"
  id="test-1-1-ss-4"
  show="report"
>
  <chat-head
    head="自由記入 ゾーイ"
  />
  <hr />
  <quill-view
    class="text quill"
    value="テスト<strong>文字列</strong>"
  />
  <!---->
  <chat-foot
    anker=""
  />
</div>
`;

exports[`Chat 4 snapshot 2`] = `
<div
  class="chat report SSAY"
  id="test-1-1-ss-4"
  show="report"
>
  <chat-head
    head="自由記入 ゾーイ"
  />
  <hr />
  <quill-input
    class="text quill"
    part_id="test-1-1"
    phases=""
  />
  <!---->
  <chat-foot
    anker=""
  />
</div>
`;

exports[`Chat 5 snapshot 1`] = `
<div
  class="chat hide SSAY"
  show="hide"
/>
`;

exports[`Chat 5 snapshot 2`] = `
<div
  class="chat hide SSAY"
  show="hide"
/>
`;

exports[`ChatFoot snapshot 1`] = `
<div
  class="date"
>
  <abbr
    class="btn"
  >
    *TEST*
  </abbr>
</div>
`;

exports[`ChatHead sign snapshot 1`] = `
<div
  class="name"
>
  <sup
    class="pull-right"
  >
    7korobi
  </sup>
  おなまえ
</div>
`;

exports[`ChatHead to snapshot 1`] = `
<div
  class="name center"
>
  <span
    class="pull-right"
  >
    おあいて
  </span>
  ▷
  <span
    class="pull-left"
  >
    おなまえ
  </span>
</div>
`;

exports[`Check off snapshot 1`] = `
<a
  class="btn"
/>
`;

exports[`Check on snapshot 1`] = `
<a
  class="btn active"
/>
`;

exports[`Check set off snapshot 1`] = `
<a
  class="btn"
/>
`;

exports[`Check set on snapshot 1`] = `
<a
  class="btn active"
/>
`;

exports[`DMode snapshot 1`] = `
<p
  style="overflow: hidden;"
>
  <span>
    <nuxt-link
      to="[object Object]"
    >
      <i
        class="mdi mdi-notebook"
      />
      メモ
      <sup>
        10
      </sup>
    </nuxt-link>
    <nuxt-link
      to="[object Object]"
    >
      バレ
      <sup>
        10
      </sup>
    </nuxt-link>
  </span>
  <span>
    <nuxt-link
      to="[object Object]"
    >
      通常
      <sup>
        10
      </sup>
    </nuxt-link>
  </span>
  <span>
    <nuxt-link
      to="[object Object]"
    >
      独り言
      <sup>
        10
      </sup>
    </nuxt-link>
    <nuxt-link
      to="[object Object]"
    >
      非日常
      <sup>
        10
      </sup>
    </nuxt-link>
    <nuxt-link
      to="[object Object]"
    >
      墓休み
      <sup>
        10
      </sup>
    </nuxt-link>
  </span>
  <span />
</p>
`;
